config/locales/migration.en.yml

```diff
@@ -3,6 +3,9 @@ en:
     campaigns:
       migration:
         form:
+          buttons:
+            add_object: Upload another file
```

Comment:
1. This appears not to be used on the form which indicates "translation missing: en.site.campaigns.migration.form.legends.add_object" and instead displays a default of "Add Object"
2. Change the text in the locale file to that on the form mockup: "Add a new object"